src/js/utils/specs/ajax-helpers-spec.js

```diff
@@ -0,0 +1,166 @@
+define(
+    [
+        'jquery',
+        'sinon',
+        '../../utils/spec-helpers/ajax-helpers.js'
+    ],
+    function($, sinon, AjaxHelpers) {
+        'use strict';
+
+        describe('AjaxHelpers', function() {
+            var testUrl = 'https://example.com',
+                testData = {foo: 'bar', fizz: 'buzz'},
+                testQuerystring = $.param(testData);
+
+            describe('fakeServer', function() {
```

Comment:
I initially planned on writing tests for these two methods but ran out of time (and/or forgot 😅). They should still probably be tested at some point though. I can:

- delete these describe blocks
- mark them up with a `TODO: implement tests` and set them up to be skip for now with `xdescribe`
- finish writing the tests

Which do you think is best?